client/state/comments/utils.js

```diff
@@ -0,0 +1,58 @@
+/***
+ * Creates a comment target id, a concatenation of siteId and postId basically
+ * @param {Number} siteId site identification
+ * @param {Number} postId post identification
+ * @returns {String} comment target id
+ */
+export function createCommentTargetId( siteId, postId ) {
+	return `${ siteId }-${ postId }`;
+}
+
+/***
+ * Creates a request id, a concatenation of siteId, postId, and query params basically
+ * @param {Number} siteId site identification
+ * @param {Number} postId post identification
+ * @param {Object} query post identification
+ * @returns {String} request id
+ */
+export function createRequestId( siteId, postId, query ) {
+	return `${siteId}-${postId}-${ JSON.stringify( query ) }`;
```

Comment:
You are absolutely right, 
I've addressed this [here](https://github.com/Automattic/wp-calypso/pull/3143/files#diff-b157f2e27a25c68197b5a64c67e94111R34).
+ added [a test](https://github.com/Automattic/wp-calypso/pull/3143/files#diff-a24dbf1e4112373bf10d54d63fbe1687R23) for that.